### ISSUE TITLE
The keyboard user is unable to turn ON the orientation radio button using Space  bar key.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#265]
### Steps to Reproduce

1.Launch Xamarin Workbook application and select platform(Mac (Modern)) and create a new workbook.
2. Navigate to File menu and invoke "Page setup". 
3. Navigate to Orientation radio button using Tab key, hit on Arrow key to switch between these buttons.
4. Try to select( Turn On) the radio buttons using the Space Bar.
5. Also observe the behaviour of the Enter key to turn On the radio button.

### Expected Behavior
The user should be able to turn ON the orientation radio button using Space  bar key. 

### Actual Behavior
 The keyboard user is unable to turn ON the orientation radio button using Space  bar key.
Note1: Its also good to turn on the same using Enter key.
Note2: The user is able to turn on the radio button using Enter key, but the problem is that, once user hit on Enter key to do the same, page setup window is also getting closed. so this is the demerit of hitting the Enter key.

### User Impact:
We know that Space bar is the default key to do the same, so user would expect the same behavior from space bar, and also by hitting on Enter key page setup window is getting closed, that is also the cause of difficulty.

### Environment

MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9)

### Log File

![kb6_pagesetup_orientationradiobutton](https://user-images.githubusercontent.com/22956130/39857276-481c27c8-5422-11e8-9664-b8d5e881c1c9.png)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS2.1.1
Xamarin-Workbook-MAC
Wipro

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1821
